### DISCUSSION TITLE
支付宝支付系统 什么时候开放 还有14天过期

### DISCUSSION THREAD

**Discussion by businiaozhimen** [getlantern/lantern-forum-cn#605] [Q&A]
支付宝支付系统 什么时候开放 还有14天过期

**Comment by MaxMKMax** [comment]
@businiaozhimen 您好. 请问您是无法用支付宝付款吗? 还是说有什么障碍?

**Reply by businiaozhimen** [reply]
![image](https://github.com/getlantern/lantern-forum-cn/assets/173088681/d94a3fd5-ca70-43d3-bca3-1521082530ae)
支付失败

**Reply by businiaozhimen** [reply]
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@businiaozhimen 好的. 我们正在解决. 第三方支付系统哪里出现问题. 我们在跟他们协调. 不好意思啊. 很抱歉影响了您的购买体验. 请您过两天再试看购买. 谢谢!
